### PR TITLE
Fixed License Contributor button [Issue #674]

### PR DESCRIPTION
**Pull request by AbhinavMir**
Solved the issue https://github.com/fossasia/fossasia.org/issues/674

View it here: https://ibb.co/2kSC5V8

### CODE REVIEW

**Review by Chaitanyassr — CHANGES_REQUESTED** [review]
please edit the respective css file.

File: licenses/debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[0212/102906.642:ERROR:crash_report_database_win.cc(428)] unexpected header
```
Comment (abishekvashok) [inline licenses/debug.log:1]: Unwanted change

File: licenses/index.html [hunk 2]
```diff
@@ -235,7 +235,7 @@ <h1>Contributor License Agreement</h1>
                                 <p>The ICLA is not tied to any employer you may have, so it is recommended to use one's personal email address in the contact details, rather than an @work address.</p>
                                 <p>The FOSSASIA PTE LTD. (201702892N) represents FOSSASIA projects, community and members in any matters, enquiries and before court.</p>
                                 <p>The FOSSASIA Contributor License Agreement and process was inspired by the APF here <a href="https://www.apache.org/licenses/">www.apache.org/licenses/</a>. In fact most parts are identical.<br/></p>
-							<a href="https://fossasia.org/cla/" class="btn btn-hollow" target="_self">Contributor License Agreement</a>
+							<a href="https://fossasia.org/cla/" class="btn btn-hollow" target="_self" style="padding:1.2vw">Contributor License Agreement</a>
```
Comment (abishekvashok) [inline licenses/index.html:238]: Don't make use of inline styles, please use the custom css file

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by mariobehling** [comment]
Missing details, missing screenshots. Therefore closing it.

**Pull request closed without merging**